scripts/python/torus_analyzer.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+# © 2022. Triad National Security, LLC. All rights reserved.  This
+# program was produced under U.S. Government contract
+# 89233218CNA000001 for Los Alamos National Laboratory (LANL), which
+# is operated by Triad National Security, LLC for the U.S.  Department
+# of Energy/National Nuclear Security Administration. All rights in
+# the program are reserved by Triad National Security, LLC, and the
+# U.S. Department of Energy/National Nuclear Security
+# Administration. The Government is granted for itself and others
+# acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works,
+# distribute copies to the public, perform publicly and display
+# publicly, and to permit others to do so.
+
+import argparse
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import sys
+import shutil
+import os
+from subprocess import call, DEVNULL
+import glob
+import time
+from enum import Enum
+from multiprocessing import Pool
+from phoebus_constants import cgs, scalefree
+import phoebus_utils
+from phoedf import phoedf
+
+from get_torus_fluxes import get_torus_fluxes
+from plot_torus_3d import plot_frame_from_phoedf
+
+# Script to perform complete analysis of an output directory
```

Comment:
this is pretty nice

Reply:
Yeah you can just call this file a bunch for different output directories in a slurm submission script and get all your analysis done in one go.